src/public/style.css

```diff
@@ -436,6 +430,84 @@ nav ul li {
   top: 0%;
   left: 0%;
   background-image: url('money.gif');
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.icon {
+  display: inline-block;
+  width: 1.5em;
+  height: 1.5em;
+  margin-left: 0.3em;
+  vertical-align: bottom;
+}
+
+.icon.money {
+  background-image: url('money.gif');
+  background-size: contain;
+  background-position: center bottom;
   background-repeat: no-repeat;
 }
 
+.icon.money:after {
+  content: none;
+}
+
+.light {
+  pointer-events: none;
+  position: absolute;
+  top: 0%;
+  left: 0%;
+  transform: translate(-50%, -50%); 
+  width: 40em;
+  height: 40em;
+  border-radius: 40em;
+  opacity: 0.4;
+  background: -moz-radial-gradient(center, ellipse cover, rgba(252,210,70,1) 0%, rgba(237,213,80,0.52) 48%, rgba(239,224,129,0) 100%);
+  background: -webkit-radial-gradient(center, ellipse cover, rgba(252,210,70,1) 0%,rgba(237,213,80,0.52) 48%,rgba(239,224,129,0) 100%);
+  background: radial-gradient(ellipse at center, rgba(252,210,70,1) 0%,rgba(237,213,80,0.52) 48%,rgba(239,224,129,0) 100%);
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#fcd246', endColorstr='#00efe081',GradientType=1 );
+  animation: pulse 30s;
+  animation-direction: alternate-reverse;
+  animation-fill-mode: both;
+  animation-iteration-count: infinite;
+  z-index: 10;
+}
+
+.light:nth-child(2) {
```

Comment:
I prefer some randomizing parameters during javascript creation instead of hard coded css.
Maybe we could find a way to use `randomiseSquare` function.

Reply:
i don't think that for 3 lights we need random parameters.
They are just here for ambiance.
But, what do you think about ?